apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -220,12 +220,14 @@ type ComponentStatus struct {
 
 // TempoStackStatus defines the observed state of TempoStack.
 type TempoStackStatus struct {
+	// Version of the Tempo Operator.
+	// +optional
+	OperatorVersion string `json:"operatorVersion,omitempty"`
+
 	// Version of the managed Tempo instance.
 	// +optional
 	TempoVersion string `json:"tempoVersion,omitempty"`
-	// Version of the Tempo Query component used.
-	// +optional
-	TempoQueryVersion string `json:"tempoQueryVersion,omitempty"`
```

Comment:
This unfortunately seems like a breaking change https://github.com/grafana/tempo-operator/blob/v0.2.0/apis/tempo/v1alpha1/tempostack_types.go#L204

Renaming/removal of a field from CRD is a braking change. 

I would keep it here marked as deprecated and kept empty.

Reply:
done